content/docs/reference/cmctl.md

```diff
@@ -6,6 +6,20 @@ description: |
 
 `cmctl` is a command line tool that can help you manage cert-manager and its resources inside your cluster.
 
+> 📢 The cert-manager CLI is moving to a new GitHub repository
+>
+> The cert-manager team have decided to move the `cmctl` code to a new GitHub repository.
+> This will allow us to release new features and bug fixes for `cmctl` independently of cert-manager.
+> It will simplify the Go package dependencies of cert-manager
+> so there should be fewer security patch releases of cert-manager.
+> It will make it easier for us to extend `cmctl` with features for managing `trust-manager` and `approver-policy`.
+> And it will allow us to write more E2E tests for `cmctl` without further slowing down the test suite of cert-manager.
+>
+> After release 1.14, `cmctl` will no longer be released with `cert-manager` itself,
```

Comment:
```suggestion
> ⚠️ cert-manager 1.14 is the last release that will still include a `cert-manager-ctl` container image, go package and GitHub release binary.
```
I tried to make this sentence a bit more concrete so it is easier to understand what will change & break.

Reply:
Also, when I read this, it is not very clear what "after release 1.14" means (eg. does that include that release?)